### ISSUE TITLE
Improve layout for space check and GPT check

### ISSUE THREAD

**Issue by htcfreek** [rcmaehl/WhyNotWin11#385]
I want to improve the checks for "GPT" and "Disk space".

### GPT
- Green = System disk is GPT
- Red = System disk is not GPT
```
System drive: GPT|MBR
Drive meets/doesn't meet requirement.
```

### Disk space
- Green = System partition and System disk >= 64 GB
- Yellow = System partition < 64 GB and System disk >= 64 GB
- Red = System partition and System Disk < 64 GB
```
System volume (C:): 250 GB (on Disk 0: 900 GB)
1/4 internal disks meet requirement.
```

---

@rcmaehl 
Are you okay with this? Then please assign me.
Who does it work with the translation changes? Dow we need an extra issue then?

**Comment by micwoj92** [comment]
I've already assigned you, just in case.

>  Who does it work with the translation changes?

I mostly approve them, but it''s the people who submit.

> Dow we need an extra issue then?

Nah, I think most translators here would see something missing and add these strings.

**Comment by rcmaehl** [comment]
I think "System volume (C:): 250 GB (on Disk 0: 900 GB)" and "1/4 internal disks meet requirement."  might be a bit long and would get cut off even without translation.

**Comment by billysielu** [comment]
I agree it's confusing so something needs to be done. Mainly the "1/2 drives" thing doesn't tell you which drive met what. Can someone explain why we're checking drives other than the system drive? Do they all need to be GPT?

There's a wider usability point that it should be possible to tell what the Windows 11 requirement is from each row of WNW11.
E.g.   Boot Method  Expected: UEFI  Actual: Legacy  Result: Fail

**Comment by htcfreek** [comment]
In general the other drive doesn't need to be checked. Because the information has not really any benefit, I will remove it. This gives me mor space for information.

**Comment by htcfreek** [comment]
Your thoughts?

![image](https://user-images.githubusercontent.com/61519853/124521424-b6e03000-ddef-11eb-8881-25e2173ef80d.png)

**Comment by micwoj92** [comment]
Looks good, do these have fixed or variable width?

**Comment by rcmaehl** [comment]
Tooltip is fine, Might need more details. For Disks I'd like to have it be like

C: # GB, Disk: # GB

Maybe make a note of how many other disks meet requirements in the tooltip for people that will do a clean install.

**Comment by rcmaehl** [comment]
> Looks good, do these have fixed or variable width?

Tooltips are semi-variable

**Comment by micwoj92** [comment]
That's good, I was afraid of some longer translations not fitting.

**Comment by htcfreek** [comment]
@rcmaehl , @micwoj92 

![image](https://user-images.githubusercontent.com/61519853/124522953-4c31f300-ddf5-11eb-8287-01ab86ee34bf.png)
_Note: This is a poc with fix written text._

**Comment by rcmaehl** [comment]
Better. My main concern is translations, but they can use shorter terms if needed.

**Comment by htcfreek** [comment]
> @htcfreek Better. My main concern is translations, but they can use shorter terms if needed.

The question is, if with shorter terms information goes lost.

**Comment by htcfreek** [comment]
How do we indicate which part of check hasn't passed if partition is to small, but didk is okay (yellow)?

**Comment by htcfreek** [comment]
work in progress

**Comment by htcfreek** [comment]
What I have so far on the feature branch in my fork:
![image](https://user-images.githubusercontent.com/61519853/124676653-3a1c8700-debf-11eb-8ea0-4b9c9cb50b54.png)

**Comment by micwoj92** [comment]
Thanks for sharing progress, you can submit pr that is marked as draft so it will be easier to check the improvements you have made.

**Comment by htcfreek** [comment]
Let me do tomorrow.

**Comment by rcmaehl** [comment]
#488 was merged manually in e6269e206d5a01e8e0e00093a33febf3211d9636
Closing
